packages/breakdown-trees/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@bentley/breakdown-trees-react",
-  "version": "0.1.2",
+  "version": "0.1.3",
```

Comment:
Hey you don't need to manually bump this

Reply:
Reverted.